src/components/Products/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+
+import './styles.scss';
+
+const Pagination = ({ pages, setCurrentPage, currentPage }) => {
+  const [limitPageNum, setLimitPageNum] = useState(6);
+  const [maxPageNumLimit , setMaxPageNumLimit] = useState(6);
+  const [minPageNumLimit , setMinPageNumLimit] = useState(1);
+
+  const handleNextPage = () => {
+    setCurrentPage(currentPage + 1);
```

Comment:
```suggestion
    setCurrentPage(prevState => prevState + 1);
```
Như này nó sẽ chặt chẽ hơn nhé